packages/client/src/object/convertWireToOsdkObjects/createOsdkObject.ts

```diff
@@ -152,11 +152,15 @@ export function createOsdkObject<
             if (instance != null) {
               return instance;
             }
+
             const geotimeProp = new GeotimeSeriesPropertyImpl<GeoJSON.Point>(
               client,
               objectDef.apiName,
               target[RawObject][objectDef.primaryKeyApiName as string],
               p as string,
+              rawValue.type === "geotimeSeriesValue"
```

Comment:
This type should only exist on the raw property value if it comes from a reference value update.